### ISSUE TITLE
Bot presence

### ISSUE THREAD

**Issue by PapyRGB** [the-cult-of-integral/discord-raidkit#81]
Hey devs !

Before asking for help, let me thank you for your incredible job!
It will be really useful for learning, but also to educate.

For my issue, I'm currently playing with the program to be familiar with its functioning, but i cannot find the way to custom my bot presence to "invisible" for example.

 I believe that it's linked to the config.json file but I don't figure out how to custom it for bot's presence.

May you help me?

**Comment by the-cult-of-integral** [comment]
I do not believe, at the moment, I have ever added something that changes the bot's presence. I will mark this issue as a feature request.

I will likely add the ability to set a bot's status during runtime via a command. I may also make it a boolean setting as to whether the bot goes invisible when performing destructive commands that do not interact with chat. It could be a config setting as to the presence the bot chooses when first starting.

Thank you for the ideas! 😄

**Comment by PapyRGB** [comment]
No problem! 😁

My point is indeed to hide it from the beginning via the config in order to make the intrusion invisible. But your two others idea are interesting too!

I will keep an eye to futures update. 👍
Until that, I will continue to play with it in my test's servers, notably to understand the process to token grabber!

Thanks in advance for the update😉

**Comment by IllegalLithium** [comment]
I have a feeling this contains some sort of trojan

**Comment by buttersmcmuffi** [comment]
it does lol

**Comment by Justman100** [comment]
@the-cult-of-integral

What here now?

**Comment by Catterall** [comment]
> I have a feeling this contains some sort of trojan

You mean to inform me that the tool that allows people to generate a trojan has the code for a trojan in it?

**Comment by the-cult-of-integral** [comment]
@PapyRGB 

I've released v2.5.0. You may find that this suggestion has finally been implemented. You can change the bot's presence now and set the bot to turn invisible when doing destructive actions.

**Comment by PapyRGB** [comment]
Thanks ! I will come back and play with this new version !
